sysdig/internal/client/secure/policies.go

```diff
@@ -67,20 +67,20 @@ func (client *sysdigSecureClient) UpdatePolicy(ctx context.Context, policyReques
 	return PolicyFromJSON(body), nil
 }
 
-func (client *sysdigSecureClient) GetPolicyById(ctx context.Context, policyID int) (policy Policy, err error) {
+func (client *sysdigSecureClient) GetPolicyById(ctx context.Context, policyID int) (policy Policy, err error, statusCode int) {
```

Comment:
```suggestion
func (client *sysdigSecureClient) GetPolicyById(ctx context.Context, policyID int) (policy Policy, statusCode int, err error) {
```

In idiomatic go, the convention is to have the error return value as the last value returned.